scripts/Start-CleanMachineTesting.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS
+	Installs Windows Sandbox feature if needed and then sets up a clean machine test enviroment
+.DESCRIPTION
+	Starts Windows Sanbox instance and runs a powershell script to setup dependencies and a repo to test gitextensions on a clean machine.
+.NOTES
+	See https://learn.microsoft.com/en-us/windows/security/application-security/application-isolation/windows-sandbox/windows-sandbox-overview
+	to learn about windows sandbox. 
+	Caveat: You must have ran a build for this to fully work.  If artifacts folder is clean, there will be errors.
+#>
+
+#required to turn on windows feature
+#requires -RunAsAdministrator
```

Comment:
Is this mandatory if WSB is installed?
I can run WSB from Start menu without UAC.

Reply:
It is if it has to enable the feature.

Reply:
I can see if I can move it to the enable step area.